templates/hackmd/0/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+hackmdPostgres:
+  image: postgres
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    POSTGRES_DB: hackmd
+
+hackmd:
+  image: hackmdio/hackmd:0.4.5
+  environment:
+    POSTGRES_USER: hackmd
+    POSTGRES_PASSWORD: hackmdpass
+    NODE_ENV: "${NODE_ENV}"
+    DEBUG: "${DEBUG}"
+    HMD_DOMAIN: "${HMD_DOMAIN}"
+    HMD_URL_PATH: "${HMD_URL_PATH}"
+    HMD_PORT: "${HMD_PORT}"
```

Comment:
as an example of hackmd lb:

in docker-compose
```
hackmdlb:
    image: rancher/lb-service-haproxy:v0.4.6
    ports:
    - 8000:8000/tcp
    labels:
      io.rancher.container.agent.role: environmentAdmin
      io.rancher.container.create_agent: 'true'
```
in rancher-compose
```
version: '2'
services:
  hackmdlb:
    scale: 1
    start_on_create: true
    lb_config:
      certs: []
      port_rules:
      - path: ''
        priority: 1
        protocol: http
        service: hackmd
        source_port: 8000
        target_port: 8000
    health_check:
      response_timeout: 2000
      healthy_threshold: 2
      port: 42
      unhealthy_threshold: 3
      interval: 2000
```
This example assumes that you set HMD_PORT to 8000, you can learn more about adding load balancer service through docker-compose from the [official documentation](https://docs.rancher.com/rancher/v1.2/en/cattle/adding-load-balancers/#adding-a-load-balancer-with-rancher-compose).